samples/VK.gs

```diff
@@ -13,9 +13,9 @@ function run() {
   var service = getService();
   if (service.hasAccess()) {
     // GET requests require access_token parameter
-    var url = 'https://api.vk.com/method/groups.get?access_token=' +
-        service.getAccessToken();
+    var url = 'https://api.vk.com/method/users.get?version=5.89&fields=first_name,last_name,email&access_token=' + service.getAccessToken()
```

Comment:
If email isn't being returned please remove it from the list of fields.

Reply:
well, that is the funny part.
If you don't add email to the list of fields, it won't be exposed on the token (`service.getToken().email `)